src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -25798,6 +25798,70 @@ public void Test1(IDo<object> impl)
             Assert.Equal("IResult<System.Int32>", doSymbol.ReturnType.ToTestDisplayString());
         }
 
+        [Fact]
+        [WorkItem(40430, "https://github.com/dotnet/roslyn/issues/40430")]
+        public void MissingTypeArgumentInBase_ValueTuple()
+        {
+            var lib_cs = @"
+public class ClassWithTwoTypeParameters<T1, T2>
+{
+}
+
+public class SelfReferencingClassWithTuple
+    : ClassWithTwoTypeParameters<SelfReferencingClassWithTuple, (string A, int B)>
```

Comment:
>(string A, int B) [](start = 64, length = 17)

It is very likely that the implemented fix will not handle a scenario when here we use ```SomeOtherMissingType<(string A, int B)>``` and that type is missing. Perhaps, even when the tuple type is present. Consider adding a test case for both scenarios.